updates/1/updates.sql

```diff
@@ -26,3 +26,37 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xBBc2AE13b23d7
 *
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF", "C", "Alchemix", "ALCX", 1614400082, NULL, "alchemix", "ALCX", "0xdBdb4d16EdA451D0503b854CF79D55697F90c8DF");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x420412E765BFa6d85aaaC94b4f7b708C89be2e2B", 4, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x420412E765BFa6d85aaaC94b4f7b708C89be2e2B", "C", "Brazilian Digital Token", "BRZ", 1552417004, NULL, "brazilian-digital-token", "BRZ", "0x420412E765BFa6d85aaaC94b4f7b708C89be2e2B");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xcca0c9c383076649604eE31b20248BC04FdF61cA", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xcca0c9c383076649604eE31b20248BC04FdF61cA", "C", "Bitmax Token", "BTMX", 1590380637, NULL, "bitmax-token", "BTMX", "0xcca0c9c383076649604eE31b20248BC04FdF61cA");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc834Fa996fA3BeC7aAD3693af486ae53D8aA8B50", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xc834Fa996fA3BeC7aAD3693af486ae53D8aA8B50", "C", "Convergence", "CONV", 1616385987, NULL, "convergence", "conv", "0xc834Fa996fA3BeC7aAD3693af486ae53D8aA8B50");
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("COPE", "Y", "Cope", "COPE", 1617077337, NULL, "cope", "COPE", "COPE"); INSERT INTO common_asset_details(asset_id, forked) VALUES("COPE", NULL);
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("CRON", "B", "Cryptocean", "CRON", 1557885600, NULL, "cryptocean", "CRON", "CRON"); INSERT INTO common_asset_details(asset_id, forked) VALUES("CRON", NULL);
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("FIDA", "Y", "Bonfida", "FIDA", 1608598800, NULL, "bonfida", "FIDA", "FIDA"); INSERT INTO common_asset_details(asset_id, forked) VALUES("FIDA", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9", "C", "FTX Token", "FTT", 1555852887, NULL, "ftx-token", "FTT", "0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9");
+*
```

Comment:
Why are you adding this? This already exists. https://github.com/rotki/rotki/blob/a180504ee5f8aeece81ab9daf07d0a3d27adec35/rotkehlchen/data/all_assets.json#L5349-L5358

You need to add a mapping in the exchange just like with all other exchanges for tokens that have ambiguous symbols like FTX does: https://github.com/rotki/rotki/blob/a180504ee5f8aeece81ab9daf07d0a3d27adec35/rotkehlchen/assets/converters.py#L732-L744


```suggestion
```